modules/role/examples/cross_account_role.tf

```diff
@@ -1,14 +1,14 @@
 provider "aws" {
-  version = "~> 1.2"
+  version = ">= 2.1.0"
```

Comment:
Should probably leave this as a pessimistic constraint (`~> 2.1`) just in case anyone copies it, they won't get unexpectedly buumped to aws provider 3.x.x